README.md

```diff
@@ -236,7 +236,9 @@ See the available options in the table below.
 | finishOn              | `null`                                | leaflet layer event to finish the drawn shape, like `'dblclick'`. [Here's a list](http://leafletjs.com/reference-1.2.0.html#interactive-layer-click). |
 | markerStyle           | `{ draggable: true }`                 | [leaflet marker options](https://leafletjs.com/reference-1.4.0.html#marker-icon) (only for drawing markers).                                          |
 | editable              | `false`                               | makes a `CircleMarker` editable like a `Circle`                                                                                                       |
-| hideMiddleMarkers   | `false`                               | hide the middle Markers in edit mode from Polyline and Polygon.                                                                                     |
+| hideMiddleMarkers     | `false`                               | hide the middle Markers in edit mode from Polyline and Polygon.                                                                                       |
+| markerEditable        | `true`                                | Markers and CircleMarkers changing to the edit mode after creation.                                                                                   |
+| continueDrawing       | `false` / `true`                      | The drawing mode is not finished if the value is `true`. It can be directly drawn a new shape. Markers and CircleMarkers are by default `true`.       |                                                                                |
```

Comment:
> The drawing mode is not finished if the value is `true`. It can be directly drawn a new shape. Markers and CircleMarkers are by default `true`.

Different suggestion:

> Draw-Mode stays enabled after finishing a layer to immediately draw the next layer. Defaults to `true` for Markers and CircleMarkers and `false` for all other layers.